### ISSUE TITLE
How to use my own loop weight in GCN?

### ISSUE THREAD

**Issue by HOLMEScdk** [rusty1s/pytorch_geometric#268]
`loop_weight = torch.full(
            (num_nodes, ),
            1 if not improved else 2,
            dtype=edge_weight.dtype,
            device=edge_weight.device)
        edge_weight = torch.cat([edge_weight, loop_weight], dim=0)
`
I noticed that in the source code of gcn_conve, the loop_weight is already defined, what if I want to use my own loop weight, it seems that I have to change these codes.

**Comment by rusty1s** [comment]
Hi, I suggest to inherit from `GCNConv` and reimplement `def norm`. This should be the easiest way.

**Comment by HOLMEScdk** [comment]
I got it, thanks for your help!
